### PR TITLE
Add discussion of RTX/FEC/RED/CN/DTMF

### LINKED ISSUE #206: Add support for explicitly disabling VAD via RTPSender
We agreed on the call that it is important for JS apps to be able
to tell the browser not to do VAD when sending.This should be a frob on
RtpSender.

### LINKED ISSUE #233: Section 5.8: handling of special payload types
http://rtcweb-wg.github.io/jsep/#rfc.section.5.8
[TODO: handling of CN, telephone-event, "red"]

### PR DESCRIPTION
**Pull request by juberti**
Fixes #206 
Fixes #233

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -3421,11 +3421,24 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+              mechanisms. DTX for such formats (e.g., Opus) is controlled via
+              fmtp parameters, as discussed in
+              <xref target="sec.voiceactivitydetection1" />.</t>
+
+              <t>For each specified "telephone-event" media format, enable DTMF
+              transmission for all supported media formats with the same
+              clockrate, as described in <xref target="RFC4733" />, Section
+              2.5.1.2. If the application attempts to transmit DTMF when using
+              a media format that does not have a corresponding telephone-event
+              format, this MUST result in an error.
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: You are missing a close </t>. Expect travis failure.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -3485,38 +3498,59 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+            Source RTP Stream, not the Redundancy RTP Stream.</t>
+
+            <t>Enable the RTCP feedback mechanisms referenced in the media
+            section for all Source RTP Streams using the specified
+            media formats. Specifically,
+            begin or continue sending the requested feedback types and
+            reacting to received feedback, as specified in
+            <xref target="RFC4585" />, Section 4.2. When sending RTCP feedback,
+            use the SSRC of an outgoing Source RTP Stream as the RTCP sender
+            SSRC; if no outgoing Source RTP Stream exists, choose a random one.
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Are there any restrictions on the random one here? Subject to the usual rules?

Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: yeah, I could include a ref here to 3550 if that would make it clearer.

Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: yeah I think that would help

**Pull request merged**